clerk-frontend/src/need-wizard/ConfirmStep.js

```diff
@@ -9,6 +9,8 @@ const timeoutPromise = (timeoutMillis) => {
   })
 }
 
+let result = [];
```

Comment:
Having a variable here makes debugging the renderAttributes function quite hard. 

I think it makes sense to replace the recursive parameter in that function with a "result", and use that to pass and return the result array.

Reply:
Done. So much easier and self explanatory.

Reply:
Just pushed this commit https://github.com/discipl/waardepapieren/pull/137/commits/77b8cda383550c88260b399c7c70bd7a741fa215
Guess now I can resolve this conversation?